tdrs-backend/tdpservice/data_files/admin.py

```diff
@@ -30,6 +30,19 @@ def queryset(self, request, queryset):
         else:
             return queryset
 
+
+class DataFileInline(admin.TabularInline):
+    """Inline model for many to many relationship."""
+
+    model = DataFile.reparse_meta_models.through
+    can_delete = False
+    ordering = ["-pk"]
```

Comment:
FYI: if your intent is to order them as submitted, they might go out of order. You might wanna look into using submission date